common/place/place_common.cc

```diff
@@ -51,7 +51,7 @@ wirelen_t get_net_metric(const Context *ctx, const NetInfo *net, MetricType type
             continue;
         if (timing_driven) {
             delay_t net_delay = ctx->predictArcDelay(net, load);
-            auto slack = load.budget - net_delay;
+            auto slack = -net_delay;
```

Comment:
@gatecat This is one of the only places I had to remove `budget` where I'm not sure I need to replace it with something else.

Reply:
Looks good - this is rarely if ever used code these days anyway (and quite possibly ripe for a cleanup)